overlay/mobile-nixos/kernel/builder.nix

```diff
@@ -322,6 +323,7 @@ stdenv.mkDerivation (inputArgs // {
 
   buildFlags = [
     kernelTarget
+    "Image.gz"
```

Comment:
Add it to `installTargets` in the kernel derivation I figure. Or maybe I'll play around with the ergonomics of building an lz4-compressed kernel.

Reply:
That's not quite enough, `installTargets` get appended after `zinstall` and just reordering them seems... fragile?

Reply:
Hm, I thought that the order didn't matter here. I'll have to think about it.